### PR TITLE
add timeout for ocic connect to server

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: Yao Zengzeng <yaozengzeng@zju.edu.cn>

### CODE REVIEW

File: cmd/client/main.go [hunk 1]
```diff
@@ -87,6 +82,11 @@ func main() {
 			Value: "/var/run/ocid.sock",
 			Usage: "Socket to connect to",
 		},
+		cli.DurationFlag{
+			Name:  "timeout",
+			Value: 10 * time.Second,
```
Comment (cyphar) [inline cmd/client/main.go]: Maybe we should make the timeout in units of seconds. Although, I'm not sure how `DurationFlag` works so maybe this isn't an issue?

**Comment by runcom** [comment]
@YaoZengzeng why is this needed? can we instead add a default timeout w/o a flag if ocic tries to connects forever?

**Comment by YaoZengzeng** [comment]
@runcom Do you mean we could just use a default timeout or no timeout ?

**Comment by cyphar** [comment]
@runcom We had a timeout before, it's just that this PR makes the timeout configurable (there was a `// TODO` comment for it).

**Comment by runcom** [comment]
> @runcom We had a timeout before, it's just that this PR makes the timeout configurable (there was a // TODO comment for it).

why would you need this in ocic I'm asking..can't there be just a default timeout hardcoded? what's the use case in having this configurable via a flag?

**Comment by rhatdan** [comment]
I think the question we really need to answer is:

Is OCIC a tool we want people to use, or is it just a test tool.  If we want to ship and use it. Then we need man pages/bash completions.  If it is just for QE and we don't plan to ship/support it, then we probably do not need something like this.

**Comment by runcom** [comment]
then, as outlined in the README, we don't need extra stuff for ocic :smile_cat:

**Comment by cyphar** [comment]
In which case, can we remove `ocic` from being installed on `make install`? That will surely clear up some of the confusion.

**Comment by rhatdan** [comment]
Want to have @mrunalp chime in to see what his intentions are.  We might want to move this to a different directory and leave it for diagnosing problems for support, when k8s is breaking.

**Comment by mrunalp** [comment]
ocic is meant to be a test client. We could address the install location later when we have kubectl integration. Also, I think that this is a useful flag to have to debug issues under load.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**